cardDatabase/templates/cardDatabase/html/view_decklist.html

```diff
@@ -159,6 +187,15 @@ <h4>Card cost distribution</h4>
                     mobile
                 );
 
+                let mainDeckCards = buildCardWrappers(cards);
```

Comment:
I think that since we refer to specific zone names elsewhere it might be worth just having this access "Main", "Main Deck", etc. as usual. I say this because this will pull from any "maybeboard" zones that people add for tech cards they're considering so it may mess up their drawing from their deck.